tests/samples/yaml/epics.ibek.yaml

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=file://schemas/ibek.schema.json
+module: epics
+definitions:
+  - name: EPICS_CA_MAX_ARRAY_BYTES
+    args:
+      - type: int
+        name: max_bytes
+        description: Max size in bytes for sending arrays over channel access
+        default: 6000000
+    env_vars:
+      - name: EPICS_CA_MAX_ARRAY_BYTES
+        value: "{{ max_bytes }}"
+
+  - name: EPICS_TS_MIN_WEST
+    args:
+      - type: int
+        name: minutes_west
+        description: Set relative time zone minutes West relative to GMT (+/-720)
+        default: 0
+    env_vars:
+      - name: EPICS_TS_MIN_WEST
+        value: "{{ minutes_west }}"
+
+  - name: dbpf
+    args:
+      - type: str
+        name: pv
+        description: Name of PV
+      - type: str
+        name: value
+        description: Value to set
+    post_ioc_init:
+      - dbpf "{{ pv }}", "{{ value }}"
+
+  - name: epicsEnvSet
```

Comment:
@thomascobb is this better for defining environment variables? Entities for specific variables then have the name hard-coded. One limitation is that all values are interpreted as strings.

Reply:
Yes, I think this is fine